homeassistant/components/yolink/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "yolink",
+  "name": "YoLink",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/yolink",
+  "requirements": ["yolink-api==0.0.4"],
```

Comment:
Please don't pin requirements in the library to an exact version. Try to be as forgiving as possible when specifying version ranges of requirements in the library. We want to avoid conflicts with other integrations that depend on the same requirements.

A minimum version is usually enough and optionally a maximum version limit for the next major version.

https://github.com/YoSmart-Inc/yolink-api/blob/20ecd51a086b0ffe87d2a7eae3c92dc62c1994b6/setup.py#L27-L28

Reply:
i changed the requirements to https://github.com/YoSmart-Inc/yolink-api/blob/main/setup.py#L27-L28, And update yolink-api to 0.0.5